### DISCUSSION TITLE
problem plotting my own ROMS model output

### DISCUSSION THREAD

**Discussion by javadbabagoli** [pydata/xarray#8070] [Q&A]
### What is your issue?

Thanks a lot for this good tool for plotting ROMS model output.

I try to use my real model output for plotting and I put my nc file here in your code: ds =.../myfile.nc')

I have tested but in all of the time it give me the exact plot similar to your figure.

I have tested and the sub setting works but all figures are the same as your plot.

Is it possible to guide me on what happened?

**Comment by welcome** [comment]
Thanks for opening your first issue here at xarray! Be sure to follow the issue template!
If you have an idea for a solution, we would really welcome a Pull Request with proposed changes.
See the [Contributing Guide](https://docs.xarray.dev/en/latest/contributing.html) for more.
It may take us a while to respond here, but we really value your contribution. Contributors like you help make xarray better.
Thank you!

**Comment by TomNicholas** [comment]
Hi, please see our [norms for raising discussions/issues](https://github.com/pydata/xarray/discussions/5404). Without a lot more detail we are unlikely to be able to help you.

**Comment by javadbabagoli** [comment]
Hi,

Ok.

I step by step, I explain you.

I use your code to plot ROMS model output.

My first question is how should I use our own .nc file of ROMS instead of using your example.

For example, you put your nc file and then plot. Now i am working on my file for plotting.

Here is your code

#open dataset
ds = xr.open_dataset('/d2/shared/TXLA_ROMS/output_20yr_obc/2001/my file.nc')

I replaced my direction and file. But, what happens?

The outputs file decrease in size('ROMS_example.nc' )and creates successfully a small file.


But, the plot is exactly the same figure you plot in your figure, not my domain. 

Thanks!
